### ISSUE TITLE
update #1

### ISSUE THREAD

**Issue by loreese** [loreese/sherlock#1]
Our group hasn't met yet this week, as we will be meeting Friday's after class to plan our work. Currently, we will be narrowing down what texts in the Sherlock Holmes series we plan on working with and defining the scope of our analysis. We want to get that narrowed down as quickly as possible so we can begin working on the files. We will also set up a preliminary schedule for when we want certain tasks accomplished by using Github and delegating workflow evenly as best we can from the start. Since we will be meeting on Friday's after class, our weekly updates will be from the previous week and it will appear as if we are a week behind even though we are not.

**Comment by djbpitt** [comment]
@loreese At our instructor’s staff meeting yesterday we came up with the idea that instead of your reporting a week behind your actual work, we should extend the deadline for your group, so that you would have time to write up your report after your meeting. Is midnight Friday an okay deadline? We’d like you to have enough time to do it carefully, while still getting it up in time for your classmates to read over the weekend.

**Comment by loreese** [comment]
Yes, Zac talked to us about that as well and before midnight is plenty of time for us to write up a report. Thanks for the slight extension!

**Comment by k-busko** [comment]
This sounds like a very interesting project that you could approach in multiple different ways. Have you guys talked about the possibility of doing a comparison of the language Doyle uses in his early work versus his later work? You could pinpoint possible shifts in his perspective of deviant bodies by noting patterns in his vocabulary that revolves around disabled people in his stories.

**Comment by engeljonathan98** [comment]
One particular story that might be of interest here is ["The Adventure of the Yellow Face"](https://en.wikipedia.org/wiki/The_Adventure_of_the_Yellow_Face) ([text here](https://etc.usf.edu/lit2go/40/the-memoirs-of-sherlock-holmes/574/adventure-2-the-yellow-face/)). The story is essentially about one of Holmes's clients hiding their daughter with some kind of "grotesque" mask that makes them seem deformed/deviant/something adjacent to that. They do this to hide the fact that this daughter is _actually_ mixed race.

I'm not sure how relevant that is to your end research goal, but it definitely strikes me as very revealing of how Doyle thought about deformity vs. how he thought about the similarly criminalized concept of race.
